src/Resources/config/services.xml

```diff
@@ -32,5 +32,10 @@
             <argument type="service" id="templating.engine.twig" />
             <argument type="service" id="knp_snappy.pdf" />
         </service>
+        
+        <service id="Sylius\InvoicingPlugin\EventListener\AdminOrderShowAfterSummaryEventListener">
```

Comment:
Couldn't you use `Sylius\Bundle\UiBundle\Block\BlockEventListener` instead of your own listener? (http://docs.sylius.com/en/1.2/customization/template.html#how-to-use-template-events-for-customizations)